File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        int answer = 0;
        
        switch(ineq + eq){
            case ">=":
                if(n>=m){
                    answer = 1;
                }else{
                    answer = 0;
                }
                break;
                
            case "<=":
                if(n<=m){
                    answer = 1;
                }else{
                    answer = 0;
                }
                break;
                
            case ">!":
                if(n>m){
                    answer = 1;
                }else{
                    answer = 0;
                }
                break;
                
            case "<!":
                if(n<m){
                    answer = 1;
                }else{
                    answer = 0;
                }
                break;
        }
        return answer;
    }
}

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.py
def solution(ineq, eq, n, m):
    answer = 0
    if ineq == ">" and eq == "=":
        if n >= m:
            return 1
        return 0
    elif ineq == ">" and eq == "!":
        if n > m:
            return 1
        return 0
    elif ineq == "<" and eq == "=":
        if n <= m:
            return 1
        return 0
    elif ineq == "<" and eq == "!":
        if n < m:
            return 1
        return 0
    